WebService/Runtime/Config.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Concurrency;
```

Comment:
as part of this PR, please import also this change: https://github.com/Azure/device-simulation-dotnet/commit/4c004d205368c371c92fc13ca64d0fa4e0250478

Reply:
Done.